manifest.json

```diff
@@ -1,30 +1,30 @@
 {
-  "serial": 2022030901,
-  "hash": "Z8RUWKYNTKuszyGv3UqhUEUkNeR7vOZPa6VVr/qRsN0=",
+  "serial": 2022031001,
+  "hash": "kuB3haut5UehpPyFz3Xvn+dc/lmm/o0A4X2d4gFx9Gs=",
   "signatures": [
     {
       "address": "13YqCei6dP2ibq2DAy81NmU8FYwwzo5HLKXFA3yM3Rvq9WxDfKF",
-      "signature": "1Dif/UYGwa5BdY/Lq1embIC15sG41zWpLW66gdNtHdI3hry92kl+Uiy5b96/lW4xJbJOXRHb4BBsvDuG10e6DQ=="
+      "signature": ""
     },
     {
       "address": "14egjs9cSxt4jnaZTYQE4NemQVXL6qtayqNNpy3VJPSS6AAcaUq",
-      "signature": "rFRjZ22dS3B18q9j3oXpI8Bp+F/i6+Hea0sp2NQUFxlT//G05KhBGvIcDW/m0AqYTwUCDXs6yV+ZBglpU5/8CQ=="
+      "signature": ""
```

Comment:
```suggestion
      "signature": "sn6O3MIoZohIN0xg2luUv+gmZyctJp/mOP4PKN/gM7MW/jhworZbTmwn8kRJaJy+f9sPwdQdlZ5A0iqVEXXkCg=="
```

Reply:
> 

Please @jcronyn consider to wait and take more time to merge this pull requests